CSS/completions/properties.py

```diff
@@ -0,0 +1,1366 @@
+from .common import get_common_values
+
+
+def get_properties():
```

Comment:
I personally don't like the idea of prefixed properties being automatically completed/suggested, but I could well be in the minority here, so it would be interesting to hear what other people think. I'd be okay with an easy way to disable them, however.

Reply:
I don't have a strong oppinion about it. Just copied what MDN offered. Removing is always an easy fix. :-D

Reply:
Removed.